### ISSUE TITLE
In 'starred' view the hide read button does not function

### ISSUE THREAD

**Issue by kalahiri** [Etuldan/spaRSS#36]
In all other views, the hide read button (the floating 'eye' down on the right) let you hide the read articles. Pressed again, it makes the read articles visible again. Only in 'starred' view nothing happens when one presses the 'eye' at the bottom of the screen. I suppose this is a bug.

**Comment by ildar** [comment]
not really. I remember Fred implemented avoid hiding the starred items.
Though I imagine someone would want to hide read items in starred view...

вс, 12 июля 2015 г. в 16:28, kalahiri <notifications@github.com>:

> In all other views, the hide read button (the floating 'eye' down on the
> right) let you hide the read articles. Pressed again, it makes the read
> articles visible again. Only in 'starred' view nothing happens when one
> presses the 'eye' at the bottom of the screen. I suppose this is a bug.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Etuldan/spaRSS/issues/36>.
>
-- 
Ildar Mulyukov,
child of God

email: ildar.mulyukov@gmail.com
GoogleTalk: ildar.mulyukov@gmail.com
blog: http://johan-notes.blogspot.com/

**Comment by kalahiri** [comment]
Thx for your reply. If that is the case, what is the use of showing the user the eye-button doing nothing.  In my opinion, either the button should be removed or the button should do it's trick. Now it seems a bit confusing.

**Comment by Etuldan** [comment]
Hi,
At the moment, I'm only focusing on bug fixes and some enhancement.

Concerning the "eye" button, I'm not using it, but it was on Flym (that I forked), and seems usefull for some people.
Maybe later, I'll rework it.
